src/java/picard/sam/ViewSam.java

```diff
@@ -75,6 +82,9 @@
     @Option(doc="Print the alignment records only.", optional = true)
     public boolean RECORDS_ONLY = false;
 
+    @Option(shortName = "int", doc = "An intervals file used to restrict what records are output.", optional = true)
+    public File INTERVAL_LIST;
```

Comment:
I would get rid of shortname here.  Also, all options should be capitalized, even the shortname.